puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-deb.groovy

```diff
@@ -16,6 +16,7 @@ pipeline {
                 script {
                     runCicoJobsInParallel([
                         ['name': 'debian9', 'job': 'foreman-nightly-debian9-test'],
+                        ['name': 'debian10', 'job': 'foreman-nightly-debian10-test'],
```

Comment:
Would this pass? We're very close to branching and adding it here probably means committing to getting it supported for 1.24. Do you think that's doable? I'd also be OK with experimental support.

cc @tbrisker?

Reply:
would it be possible to have it as experimental without blocking releases if it fails? if not i'd rather wait for 1.25 with it

Reply:
The built packages worked fine locally in a vagrant VM, so I think if something fails, it would be something small, but I'm also fine with with just adding it first and run it manually to see, where it's going.

Reply:
makes sense to me, let's try building it manually for for now and if we feel it's stable enough we can still add it to the release pipelines in time for 1.24 GA - just so we don't get blocked in releasing RCs because something is still missing.

Reply:
deleted.

Reply:
To be clear, I think this particular line is fine for now and if it worked in a local vagrant machine, then we should keep it in here.

Reply:
OK, re-added... we'd need to see if it works in that particular setup anyway....

Reply:
You should also re-add the `Push DEBs` step.